elected_targets/models.py

```diff
@@ -49,7 +49,7 @@ def title_abbrev(self):
 
     def target_name(self):
         target = getattr(self, self.target_column, '')
-        return re.sub(r'(_|-0+)', '-', target)
+        return re.sub(r'(_0+|-0+|_)', '-', target)
```

Comment:
This kind of regex scares me a little -- can we add a test or two and add a documentation line saying what it's doing.